### PR TITLE
Poster image with audio

### PR DESCRIPTION
**Pull request by simondate**
Makes sense as a feature. I remember we did this at Kineo. Double check for both the mp3 file and for the poster so nothing will change unless the user is searching for it

[#2264](https://github.com/adaptlearning/adapt_framework/issues/2264)

### CODE REVIEW

File: templates/media.hbs [hunk 1]
```diff
@@ -5,6 +5,12 @@
     <button class="aria-label js-skip-to-transcript" aria-label="{{_globals._components._media.skipToTranscript}}"></button>
     {{/any}}
     <div class="media-widget component-widget a11y-ignore-aria">
+    
+    {{#if _media.mp3}}
+      {{#if _media.poster}}
```
Comment (moloko) [inline templates/media.hbs]: this line and the following two could be moved inside the existing `{{#if _media.mp3}}` on L15

**Comment by lc-thomasberger** [comment]
Is there a related issue?

File: templates/media.hbs (hunk 1 above)
Comment (simondate) [reply templates/media.hbs]: Haha yes, that would be simpler. Sorry I was working quite late and didn't really what I was doing.

**Comment by simondate** [comment]
@lc-thomasberger I can create one if it's required. I think it's quite self-explanatory though?

**Comment by oliverfoster** [comment]
Issue > PR is the standard workflow at the moment. It helps us keep track of what's needed and what is going on.

**Comment by moloko** [comment]
https://github.com/adaptlearning/adapt_framework/wiki/Contributing-code#before-submitting

**Comment by simondate** [comment]
https://github.com/adaptlearning/adapt_framework/issues/2264

**Comment by oliverfoster** [comment]
ta

**Comment by brian-learningpool** [comment]
Indentation was already a bit inconsistent in the .hbs, sometimes 2 sometimes 4 spaces.  I don't think @simondate's PR has introduced anything in regard to this?  Are we ok to merge this and address the indentation inconsistencies as a separate housekeeping task?

**Pull request merged**